doc/_static/page_history.js

```
@@ -362,12 +314,13 @@ function get_history(name, lang) {
         validlang = lang;
 
         if (versionsArr[i] < data.newat) break;
+        if (data.removedat != '' && versionsArr[i] > data.newat) break;
 
         /* for zh-Hans non translated versions use english */
-        validlang = (lang == 'zh-Hans' && versionsArr[i] == '3.7')? lang : 'en';
+        validlang = (lang == 'zh-Hans' && versionsArr[i] >= '3.7')? 'zh_Hans' : 'en';
 
         var link = home + versionsArr[i] + '/';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)